docs/source/cli/certs.rst

```diff
@@ -106,6 +106,25 @@ You can also disable adding the CommonName as ``subjectAltName``:
 
 ... this will only have "example.net" but not example.com as ``subjectAltName``.
 
+Advanced subject alternative names
+=============================
```

Comment:
You need to fix the nunber of `=` to match the header above. That is what the docs build is  complaining about. ReStructuredText ist annoyingly picky here.

Reply:
Ah, thanks for the heads-up. I usually use `#`-syntax for headers.

Reply:
Sphinx has something you might call a pseudo standard here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
*sigh* https://xkcd.com/927/

Reply:
I don't know of any other standard regarding this.